plugins/module_utils/redfish_utils.py

```diff
@@ -91,12 +117,36 @@ def post_request(self, uri, pyld):
         req_headers = dict(POST_HEADERS)
         username, password, basic_auth = self._auth_params(req_headers)
         try:
-            resp = open_url(uri, data=json.dumps(pyld),
+            session_id = ''
+            if "iLOlogin" in pyld.keys() or os.path.isfile(HOME + "/sessionfile" + self.root_uri[8:] + ".txt"):
```

Comment:
I'm concerned there's product name in the generic Redfish code here. Things like this should be left to OEM extensions.

Reply:
Ok, we will get it removed.

Reply:
This is removed as part of latest commit.